### DISCUSSION TITLE
Ignore specific repairs

### DISCUSSION THREAD

**Discussion by noxhirsch** [frenck/spook#316] [Ideas]
I hate open repairs and the associated red circled number in the frontend ;) Therefore I always update (or skip) immediately when HACS offers me an update. But since I don't want to reboot all the time, but only when the update is important for me, I use the Ignore All feature of Spook to get rid of the reboot repairs.
I'd like to automate that - but for that it would make sense to have a service to ignore only certain issue_ids to avoid "collateral damage".

**Comment by frenck** [comment]
Finding the right issue IDs can be hard (especially when they are dynamic 😬 ).

How about existing the ignore all with a `domain` parameter? Would that work?

../Frenck

**Reply by noxhirsch** [reply]
> Finding the right issue IDs can be hard (especially when they are dynamic 😬 ).

You're right about that. The only way I can think of, is to catch the event and use `trigger.event.data.issue_id`. But this is not the most user friendly solution.

> How about existing the ignore all with a domain parameter? Would that work?

I think that's an easy (for users) and great solution 👍 In my example you could then just ignore all repairs with `domain: hacs`.

**Comment by frenck** [comment]
> The only way I can think of, is to catch the event and use trigger.event.data.issue_id. But this is not the most user friendly solution.

Or use the `event.repairs` entity Spook provides 😉 

>  In my example you could then just ignore all repairs with domain: hacs.

Exactly 👍 

../Frenck

**Reply by noxhirsch** [reply]
> Or use the event.repairs entity Spook provides 😉

Stupid me, yes of course!🤦‍♂️ I haven't had the time to look into event entities yet - so I forgot about them. But they are definitely on my to-do list :)
